### PR TITLE
Ct 788 build ie lib in monorepo

### PR DESCRIPTION
**Pull request by romanbracinik**
Pridavam PR na build pre IE libku, testy presli tu. 

https://github.com/keboola/storage-backend/actions/runs/4232059270

Prev PR pridava len tu libku IE do monorepa https://github.com/keboola/storage-backend/pull/36

### CODE REVIEW

**Comment by romanbracinik** [comment]
Info do tabulky pridavam tu https://github.com/keboola/storage-backend/pull/43 blbo sa to rebasuje :

File: composer.json [hunk 1]
```diff
@@ -12,40 +12,59 @@
...
         "squizlabs/php_codesniffer": "^3",
+        "symfony/finder": "^5.4",
         "symplify/monorepo-builder": "11.1.30.72"
     },
     "autoload": {
         "psr-4": {
-            "Keboola\\StorageBackend\\": "src/",
             "Keboola\\Datatype\\": "packages/php-datatypes/src/",
+            "Keboola\\Db\\ImportExport\\": "packages/php-db-import-export/src/",
+            "Keboola\\Provisioning\\": "packages/php-db-import-export/provisioning/./src",
```
Comment (jirkasemmler) [inline composer.json:46]:
```suggestion
            "Keboola\\Provisioning\\": "packages/php-db-import-export/provisioning/src",
```

Comment (romanbracinik) [reply composer.json:46]: Divne to vygeneroval composer

Comment (jirkasemmler) [reply composer.json:46]: tak klido nech, jen jsem si vsiml

**Review by jirkasemmler — APPROVED** [review]
stejne jak v #36 - nainstaloval jsem si to a pustil test->cajk. jen prosim nekde zdokumentuj ten autoload

**Pull request merged**